src/lib.rs

```diff
@@ -1999,6 +1999,15 @@ pub trait Itertools: Iterator {
     /// assert_eq!(numbers.iter().find_or_last(|&&x| x > 5), Some(&4));
     /// assert_eq!(numbers.iter().find_or_last(|&&x| x > 2), Some(&3));
     /// assert_eq!(std::iter::empty::<i32>().find_or_last(|&x| x > 5), None);
+    ///
+    /// // An iterator of Results can return the first Ok or the last Err:
+    /// let input = vec![Err(()), Ok(11), Err(()), Ok(22)];
+    /// assert_eq!(.into_iter().find_or_last(Result::is_ok), Some(Ok(11)));
```

Comment:
Should this be `assert_eq!(input.into_iter().find_or_last(Result::is_ok), Some(Ok(11)));`?